src/Components/SigDetailsTable/SigDetailsTable.js

```diff
@@ -169,19 +168,15 @@ ${host.matches.length > 1 && key !== host.matches.length - 1 ? `~~~~~~~~~~~~~~~~
     }, [intl, data]);
 
     useEffect(() => {
-        // populates the Group filter list
+        // populates the Group name filter dropdown
         const getGroupList = groupsData => {
-            return groupsData?.filter(host => !isEmpty(host.groups)).map(host => {
-                const group = host.groups[0];
-                return {
-                    label: group.name,
-                    value: JSON.stringify({ id: group.id, name: group.name })
-                };
-            });
+            return groupsData?.filter(host => !isEmpty(host.groups)).map(host => (
+                { label: host.groups[0].name, value: host.groups[0].name })
+            );
         };
 
         let groupsList = getGroupList(groupsData?.rulesList[0]?.affectedHostsList) || [];
-        groupsList = uniqWith(groupsList, (group1, group2) => group1.label === group2.label);
+        groupsList = uniqWith(groupsList, (group1, group2) => group1.label === group2.label); // remove duplicate group names
```

Comment:
If you wanna do some magic here and avoid using the lodash helper, you can do:

```js
const groupsList = Array.from(new Set(getGroupList(groupsData?.rulesList[0]?.affectedHostsList) || []))
```

The `Set` will also remove duplicates, as it allows an item to only be present once[0].


[0] https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set#description

Reply:
Yeah, I tried using that but it didn't seem to remove the duplicate entries in the list. /shrug  Oh well, I'll stick with lodash for the time being.  Thanks.